.travis.yml

```diff
@@ -22,25 +26,31 @@ matrix:
     fast_finish: true
     include:
         - php: 5.3
-          env: SYMFONY_DEPRECATIONS_HELPER=weak
+          dist: precise
         - php: 5.3
-          env: COMPOSER_FLAGS="--prefer-lowest" SYMFONY_DEPRECATIONS_HELPER=weak
+          dist: precise
+          env: SYMFONY_DEPRECATIONS_HELPER COMPOSER_FLAGS="--prefer-lowest"
         - php: 5.4
-          env: SYMFONY_DEPRECATIONS_HELPER=weak
         - php: 7.0
-          env: SYMFONY_VERSION=2.7.* SYMFONY_DEPRECATIONS_HELPER=weak
+          env: SYMFONY_DEPRECATIONS_HELPER=weak_vendors SYMFONY_VERSION=2.7.*
         - php: 7.0
-          env: SYMFONY_VERSION=2.8.* COVERAGE=true SYMFONY_DEPRECATIONS_HELPER=weak
+          env: SYMFONY_DEPRECATIONS_HELPER=weak_vendors SYMFONY_VERSION=2.8.* COVERAGE=true
         - php: 7.0
-          env: SYMFONY_VERSION=3.0.*
+          env: SYMFONY_DEPRECATIONS_HELPER=weak_vendors SYMFONY_VERSION=3.2.*
+        - php: 7.1
+          env: ~
+    allow_failures:
+        - php: 7.1
+          env: ~
 
 before_script:
-    - if [ "$COVERAGE" != "true" ] && [ "$TRAVIS_PHP_VERSION" != "hhvm" ]; then phpenv config-rm xdebug.ini; fi
+    - if [ "$COVERAGE" != "true" ]; then phpenv config-rm xdebug.ini; fi
```

Comment:
Since we are collecting coverage under PHP 7.0, can we use PHPDBG as a driver and drop xDebug altogether? IT will make the build faster

Reply:
Feel free to do it, I just want to make the build green, and have no idea how to do what you are talking about.

Reply:
Roger, will do it with a separate PR when this will be merged.